src/views/Create.vue

```diff
@@ -197,14 +195,23 @@ export default {
 			return this.form.questions && this.form.questions.length === 0
 		},
 
-		/**
-		 * Check if at least one question is mandatory
-		 * @returns {Boolean}
-		 */
-		mandatoryUsed() {
-			return this.form.questions.reduce(
+		infoMessage() {
+			const isMandatoryUsed = this.form.questions.reduce(
 				(isUsed, question) => isUsed || question.mandatory
 				, false)
+			if (isMandatoryUsed) {
+				if (this.form.isAnonymous) {
+					return t('forms', 'Responses are connected to your Nextcloud account. An asterisk (*) indicates mandatory questions.')
+				} else {
+					return t('forms', 'Responses are anonymous. An asterisk (*) indicates mandatory questions.')
+				}
+			} else {
+				if (this.form.isAnonymous) {
+					return t('forms', 'Responses are connected to your Nextcloud account.')
+				} else {
+					return t('forms', 'Responses are anonymous.')
+				}
+			}
```

Comment:
You can use string combination to avoid making translators write 4 different translations :)

```suggestion
			let message = ''
			if (this.form.isAnonymous) {
				message += t('forms', 'Responses are connected to your Nextcloud account.')
			} else {
				message += t('forms', 'Responses are anonymous.')
			}
			if (this.isMandatoryUsed) {
				message += ' ' + t('forms', 'An asterisk (*) indicates mandatory questions.')
			}
			return message
```

Also, are you sure about `this.form.isAnonymous` ?
shouldn't we return `Responses are anonymous` when it's true instead? Missing a `!` ?

Reply:
It's my mistake. Thanks!!